src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -335,6 +343,15 @@ internal static MemoryInfoResult GetMemoryInfo(SmapsReader smapsReader = null, b
             }
         }
 
+        public static long GetTotalScratchAllocatedMemory()
```

Comment:
This is wrong. You are computing the total size of the scratches, but you need the *dirty* size of them.
We already can tell that from the scrtaches themselves, so trying to do this roundabout way is actually more complex.
Have a way to find all the live scratches and ask them for their used mem